frontend/landmark-identifier/style.css

```
@@ -423,73 +628,257 @@
+}
+
+/* Print Styles */
+@media print {
+  .landmark-hero, .upload-card, .history-section, .landmark-footer, .btn-primary, .btn-secondary {
+    display: none !important;
+  }
+  
+  .result-card {
+    border: 1px solid #000;
+    box-shadow: none;
+    page-break-inside: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)